webapp/package-lock.json

```diff
@@ -1044,9 +1044,9 @@
       },
       "dependencies": {
         "minimist": {
-          "version": "0.2.2",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.2.2.tgz",
-          "integrity": "sha512-g92kDfAOAszDRtHNagjZPPI/9lfOFaRBL/Ud6Z0RKZua/x+49awTydZLh5Gkhb80Xy5hmcvZNLGzscW5n5yd0g=="
+          "version": "0.2.4",
```

Comment:
is `package-lock.json` file commit intentional?

Reply:
nope, thanks for flagging.